modules/meeting/app/forms/meeting/start_date.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+class Meeting::StartDate < ApplicationForm
+  form do |meeting_form|
+    meeting_form.text_field(
+      name: :start_date,
+      type: "date",
+      value: @initial_date,
+      placeholder: @meeting.class.human_attribute_name(:start_date),
+      label: @meeting.class.human_attribute_name(:start_date),
+      required: true,
+      autofocus: false,
+      data: {
+        action: "input->recurring-meetings--form#updateFrequencyText input->meetings--form#updateTimezoneText"
```

Comment:
I like putting these on multiple lines, I find it easier to scan. But that's a preference. It's perfectly ok as is.

```suggestion
        action: "input->recurring-meetings--form#updateFrequencyText" \
                "input->meetings--form#updateTimezoneText"
```